doc/libpmemobj/pmemobj_action.3.md

```diff
@@ -194,6 +202,11 @@ reserved object, otherwise an *OID_NULL* is returned.
 On success, **pmemobj_tx_publish**() returns 0, otherwise,
 stage changes to *TX_STAGE_ONABORT* and *errno* is set appropriately
 
+On success, **pmemobj_tx_xpublish**() returns 0, otherwise, when the
+**POBJ_XPUBLISH_NO_ABORT** flag is enabled the error number is returned,
+when the flag is disabled, the stage is changed to *TX_STAGE_ONABORT*,
+and *errno* is set appropriately.
```

Comment:
This description doesn't say what happens when POBJ_XPUBLISH_NO_ABORT is not set.